src/widgets/smart-home/entity-state/entity-state.widget.tsx

```diff
@@ -101,6 +111,7 @@ function EntityStateTile({ widget }: SmartHomeEntityStateWidgetProps) {
     dataComponent = (
       <Text align="center">
         {data?.state}
+        {widget.properties.attributeValue && data?.attributes.unit_of_measurement? " " + data?.attributes.unit_of_measurement: ""}
```

Comment:
Can you extract this to a const instead of doing it in the JSX?

Reply:
Sure! Do you want the changes as a new commit or do you want me to squash the existing commit?

Reply:
Please amend to the existing commit. Pull requests should only contain a single commit.
Thank you 👍